mitodl/hugo-course-publisher
# MathJax support

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/4YAErbkD/61-mathjax-support

#### What's this PR do?
This PR adds the `mathjax@3` npm package, which includes the pre-built es5 components of MathJax.  Since at the moment we are simply loading TeX markup from the markdown files, I simply used a `CopyWebpackPlugin` statement to copy the JS to Hugo's static folder and load it using [this](http://docs.mathjax.org/en/latest/web/configuration.html#configuring-and-loading-in-one-script) method.

#### How should this be manually tested?
Run the site, visit some courses that use MathJax and verify that equations are properly displayed in both inline and block formats.  Some courses with MathJax are:

 - 16-121-analytical-subsonic-aerodynamics-fall-2017
 - 18-s997-introduction-to-matlab-programming-fall-2011
 - 15-071-the-analytics-edge-spring-2017/sections/logistic-regression

#### Screenshots (if appropriate)
![Screen Shot 2020-07-16 at 2 53 12 PM](https://user-images.githubusercontent.com/12089658/87710741-2939a980-c774-11ea-9fe1-4ad3f2b408b7.png)
![Screen Shot 2020-07-16 at 2 53 24 PM](https://user-images.githubusercontent.com/12089658/87710748-2b9c0380-c774-11ea-846e-ab8bcfffc30f.png)
